usaspending_api/reporting/v2/views/agencies/overview.py

```diff
@@ -106,17 +108,24 @@ def get_agency_overview(self):
                 "abbreviation",
                 "toptier_code",
                 "total_dollars_obligated_gtas",
-                "total_budgetary_resources",
-                "total_diff_approp_ocpa_obligated_amounts",
+                "current_total_budget_authority_amount",
+                "obligation_difference",
                 "recent_publication_date",
                 "recent_publication_date_certified",
                 "tas_obligations",
                 "tas_obligation_not_in_gtas_total",
-                "missing_tas_accounts",
+                "missing_tas_accounts_count",
+                "fiscal_year",
+                "fiscal_period",
+                "missing_tas_accounts_count",
                 "fiscal_year",
                 "fiscal_period",
                 "submission_is_quarter",
```

Comment:
Several duplicate values in this list

Reply:
That's what happens when you don't look at merge conflicts that closely